examples/gallery/Cargo.toml

```diff
@@ -22,13 +22,13 @@ slint-build = { path = "../../api/rs/build" }
 
 # Remove the `#wasm#` to uncomment the wasm build.
 # This is commented out by default because we don't want to build it as a library by default
-# The CI has a script that does sed "s/#wasm# //" to generate the wasm build.
+# The CI has a script that does sed "s///" to generate the wasm build.
```

Comment:
you did not want to commit this change! :-)

Reply:
oh yes, thank you